src/stores/Object.js

```diff
@@ -11,6 +11,7 @@ const concurrentUploads = 3;
 
 class ObjectStore {
   @observable writeTokens = {};
+  @observable forceUpdate = 0;
```

Comment:
is this forceUpdate thing really necessary? I'm guessing it's not updating properly when nested fields of the observable are changed?

if you can't find a better way to deal with it, I would recommend putting the force update flag in the component calling the updating method, if possible

Reply:
Removed the store-level `forceUpdate` and used component state for the update. I swear this is something I tried and didn't work. Well it works now.